src/templates/users/user_detail.html

```diff
@@ -3,25 +3,43 @@
 {% load static %}
 
 {% block title %}
-  User:
-  {{ object.name }}
+  My Details:
 {% endblock title %}
 {% block content %}
   <div class="container">
     <div class="row">
       <div class="col-sm-12">
-        <h2>{{ object.name }}</h2>
+        <h2>My Details:</h2>
```

Comment:
I think My info and My profile will be confusing to the user. Can you explain the difference between them?

Reply:
So my info will just display all their information, and can only be accessed that user. It's not like a profile where any user can access it.